src/pages/Home/index.tsx

```diff
@@ -1,5 +1,11 @@
+import SideBar from './ui/SideBar';
+
 const Home = () => {
-  return <main></main>;
+  return (
+    <main>
+      <SideBar></SideBar>
```

Comment:
셀프 클로징 태그 형식으로 작성해 주세요.
```suggestion
      <SideBar />
```